pkgs/applications/display-managers/greetd/qtgreet.nix

```diff
@@ -17,7 +17,7 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "qtgreet";
-  version = "2.0.3.95";
+  version = "2.0.4";
```

Comment:
Just updating the version isn't sufficient, you also have to update `src.hash`

Reply:
On second look, there doesn't even seem to be a 2.0.4 tag on the upstream repo?

Reply:
Indeed, both point to the same commit, as 2.0.3.95 was a release candidate for 2.0.4.